billy-france/src/test/java/com/premiumminds/billy/france/test/services/export/TestFRCreditReceiptPDFTransformer.java

```diff
@@ -82,14 +83,19 @@ public void setUp() throws FileNotFoundException {
     }
 
     @Test
-    public void testPdfCreation() throws ExportServiceException,
-            DocumentIssuingException, IOException {
+    public void testPdfCreation() throws
+								  ExportServiceException,
+								  DocumentIssuingException,
+								  IOException,
+								  SeriesUniqueCodeNotFilled,
+								  DocumentSeriesDoesNotExistException
+	{
```

Comment:
This portion of code doesn't follow the style guidelines. Make sure you use the given code formatter